interop/configs/commit.json

```diff
@@ -0,0 +1,128 @@
+      {"action": "groupContextExtensionsProposal", "actor": "alice1", "extensions": [
+        {"extension_type": 123, "extension_data": "aGVsbG8="},
+        {"extension_type": 321, "extension_data": "d29ybGQ="}
+      ]},
+      {"action": "fullCommit", "actor": "alice2", "byReference": [8], "members": ["alice1", "alice3"]}
+    ],
+
+    "external_psk": [
+      {"action": "createGroup", "actor": "alice"},
+      {"action": "externalJoin", "actor": "alice", "joiner": "bob"},
+      {"action": "installExternalPSK", "clients": ["alice", "bob"]},
+      {"action": "installExternalPSK", "clients": ["alice", "bob"]},
+
+      {"action": "externalPSKProposal", "pskID": 2, "actor": "alice"},
+      {"action": "externalPSKProposal", "pskID": 3, "actor": "bob"},
+      {"action": "fullCommit", "actor": "alice", "byReference": [4, 5], "members": ["bob"], "force_path": true},
+
+      {"action": "externalPSKProposal", "pskID": 2, "actor": "alice"},
+      {"action": "externalPSKProposal", "pskID": 3, "actor": "bob"},
+      {"action": "fullCommit", "actor": "alice", "byReference": [4, 5], "members": ["bob"], "force_path": false}
+    ],
+
+    "resumption_psk": [
+      {"action": "createGroup", "actor": "alice"},
+      {"action": "externalJoin", "actor": "alice", "joiner": "bob"},
+      {"action": "fullCommit", "actor": "alice", "members": ["bob"]},
+
+      {"action": "resumptionPSKProposal", "epochID": 1, "actor": "alice"},
+      {"action": "resumptionPSKProposal", "epochID": 1, "actor": "bob"},
+      {"action": "fullCommit", "actor": "alice", "byReference": [3, 4], "members": ["bob"], "force_path": true},
+
+      {"action": "resumptionPSKProposal", "epochID": 2, "actor": "alice"},
+      {"action": "resumptionPSKProposal", "epochID": 2, "actor": "bob"},
+      {"action": "fullCommit", "actor": "alice", "byReference": [3, 4], "members": ["bob"], "force_path": false}
+    ],
+
+    "all_together": [
+      {"action": "createGroup", "actor": "alice"},
+      {"action": "externalJoin", "actor": "alice", "joiner": "bob"},
+      {"action": "createKeyPackage", "actor": "charlie1"},
+      {"action": "createKeyPackage", "actor": "charlie2"},
+      {"action": "addProposal", "actor": "alice", "keyPackage": 2},
+      {"action": "addProposal", "actor": "alice", "keyPackage": 3},
+      {"action": "fullCommit", "actor": "alice", "byReference": [4, 5], "members": ["bob"], "joiners": ["charlie1", "charlie2"]},
+      {"action": "installExternalPSK", "clients": ["alice", "bob"]},
+
+      {"action": "createKeyPackage", "actor": "charlie3"},
+      {"action": "addProposal", "actor": "alice", "keyPackage": 8},
+      {"action": "removeProposal", "actor": "alice", "removed": "charlie1"},
+      {"action": "groupContextExtensionsProposal", "actor": "alice", "extensions": [
+        {"extension_type": 123, "extension_data": "aGVsbG8="},
+        {"extension_type": 321, "extension_data": "d29ybGQ="}
```

Comment:
We should normalize on either `camelCase` or `snake_case` for script parameters.  Right now, I think we're largely using `camelCase` for script parameters and `snake_case` for transcript field names.  Maybe we should just use the same for both.

Reply:
In this case how about a separate PR that makes this consistent?